pages/app/settings.vue

```diff
@@ -162,20 +169,28 @@ const onSelectProfilePhoto = (event) => {
     event.target.value = '' // Reset the input
     return
   }
-
+  imagePreview.value = URL.createObjectURL(file)
   storeUser.setProfilePic(file)
 }
 
 const onClickUploadPhoto = () => {
   profilePhotoInput.value.click()
 }
 
+const onClickClearPhoto = async () => {
```

Comment:
Error was not handled. When you have more then one async operations in the block. you should use `try { } catch () {}`, other wise gracefully catch the error using `then().catch()`